.nextcloudignore

```diff
@@ -1,29 +1,29 @@
-AUTHORS.md
-build
-.git
-.gitignore
-.travis.yml
-.tx
-.scrutinizer.yml
-CONTRIBUTING.md
-composer.json
-composer.lock
-composer.phar
-karma.conf.js
-krankerl.toml
-l10n/no-php
-Makefile
-nbproject
-node_modules
-package.json
-package-lock.json
-screenshots
-src
-stylelint.config.js
-tests
-vendor/bin
-webpack.common.js
-webpack.dev.js
-webpack.prod.js
+/AUTHORS.md
+/build
+/.git
+/.gitignore
+/.travis.yml
+/.tx
+/.scrutinizer.yml
+/CONTRIBUTING.md
+/composer.json
+/composer.lock
+/composer.phar
+/karma.conf.js
+/krankerl.toml
+/l10n/no-php
+/Makefile
+/&nbproject
+/node_modules
+/package.json
+/package-lock.json
+/screenshots
+/src
+/stylelint.config.js
+/tests
```

Comment:
this might be the exception. there are some composer dependencies that ship tests. dropping those files might be desirable.

Reply:
Added a separate rule for `/vendor/**/tests` to be more explicit about that case